tshirts/tshirts.cs

```diff
@@ -1,22 +1,31 @@
-﻿using System;
 using System.Diagnostics;
 
-namespace TshirtSpace {
-    class Tshirt {
-        static string Size(int cms) {
-            if(cms < 38) {
+namespace TshirtSpace
+{
+    class Tshirt
+    {
+        static string Size(int cms)
+        {
+            if (cms <= 38)
+            {
                 return "S";
-            } else if(cms > 38 && cms < 42) {
+            }
+            else if (cms >= 39 && cms <= 42)
+            {
                 return "M";
-            } else {
+            }
+            else
+            {
                 return "L";
             }
         }
-        static void Main(string[] args) {
+        private
+        static void Main(string[] args)
+        {
             Debug.Assert(Size(37) == "S");

```

Comment:
the values 38,39,42 are boundary values. Test cases for these values are a must at least.
Please add these tests to make your production code robust against future accidental changes which might alter the existing defined behavior.